tests/cypress/plugins/imageGenerator/index.d.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+declare namespace Cypress {
+    interface Chainable<Subject> {
+        /**
+         * @example
+         * cy.imageGenerator('./fixtures', 'image.png', 500, 500, 'gray', 'Hello world!')
+         */
+        downloadFile(directory: string, fileName: string, width: number, height: number, color: string, message: string): Chainable<any>
+    }
+}
```

Comment:
Why we need TypeScript declaration file if all tests are written on JavaScript?

Reply:
Yes, this file is not needed. Removed.